### PR TITLE
dark and light mode supported logo in readme

### LINKED ISSUE #95: [Docs] Support dark mode with light version of Service Mesh Patterns logo
**Current State:**
This repo's [readme](https://github.com/service-mesh-patterns/service-mesh-patterns#readme) doesn't consider for dark vs light mode.

**Desired State:**
Similar to other repos' readmes, include dark mode and light mode logos.

Find logos in the brand kits here: https://layer5.io/brand.

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Description**
Readme now supports both dark and light mode logo
image
image

This PR fixes #95 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by UtkarshMishra12 — DISMISSED** [review]
LGTM Thanks @sudhanshutech 🥇🚀

File: README.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 <h1 align="center">Service Mesh Patterns</h1>
 </div>
 <p align="center">
-    <img width="60%" src=".github/readme/images/logo_with_text.svg">
+<picture align="center">
+<source media="(prefers-color-scheme: dark)" srcset=".github/readme/images/service-mesh-patterns-side_white.svg" width="70%" align="center" style="margin-bottom:20px;">
+<source media="(prefers-color-scheme: light)" srcset=".github/readme/images/logo_with_text.svg" width="70%" align="center" style="margin-bottom:20px;">
```
Comment (tomkerkhove) [inline README.md:7]: Nit - You might be more consistent with file naming

Comment (sudhanshutech) [reply README.md:7]: sure I will keep that in mind

Comment (tomkerkhove) [reply README.md:7]: This is what I would suggest:
```suggestion
<source media="(prefers-color-scheme: dark)" srcset=".github/readme/images/logo_with_text_white.svg" width="70%" align="center" style="margin-bottom:20px;">
<source media="(prefers-color-scheme: light)" srcset=".github/readme/images/logo_with_text.svg" width="70%" align="center" style="margin-bottom:20px;">
```

Optionally rename `logo_with_text.svg` to `logo_with_text_black.svg` as well if it's not breaking anything.

Comment (sudhanshutech) [reply README.md:7]: Sure I will do that.

Comment (sudhanshutech) [reply README.md]: @tomkerkhove I have made the necessary changes, please have a look

File: README.md [hunk 2]
```diff
@@ -2,7 +2,10 @@
 <h1 align="center">Service Mesh Patterns</h1>
 </div>
 <p align="center">
-    <img width="60%" src=".github/readme/images/logo_with_text.svg">
+<picture align="center">
+<source media="(prefers-color-scheme: dark)" srcset=".github/readme/images/logo_with_text_white.svg" width="70%" align="center" style="margin-bottom:20px;">
+<source media="(prefers-color-scheme: light)" srcset=".github/readme/images/logo_with_text_black.svg" width="70%" align="center" style="margin-bottom:20px;">
+<img alt="Shows an illustrated light mode mesh pattern logo in light color mode and a dark mode mesh pattern dark color mode." src=".github/readme/images/logo_with_text.svg" width="70%" align="center" style="margin-bottom:20px;">
```
Comment (tomkerkhove) [inline README.md:8]: This also has to be updated
```suggestion
<img alt="Shows an illustrated light mode mesh pattern logo in light color mode and a dark mode mesh pattern dark color mode." src=".github/readme/images/logo_with_text_black.svg" width="70%" align="center" style="margin-bottom:20px;">
```

Comment (sudhanshutech) [reply README.md:8]: Oh, I am really sorry, I missed that.

Comment (tomkerkhove) [reply README.md]: No worries - That's what reviews are for

**Review by tomkerkhove — APPROVED** [review]

**Review by UtkarshMishra12 — APPROVED** [review]

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**